### PR TITLE
2.13.0

### PR DESCRIPTION
**Pull request by lusbenjamin**
This is our first experiment with an integration-branch! Please everyone feel free to edit and contribute to this.

### Also Included Feature PRs:
This list includes PRs that were part of the orginal branch and so will not get a pretty-looking squash-merge.
- #669
- #694
- #698
- #693

### TODO:
- [x] fix card hover on Economy page
- [x] merge in ~~#684~~ #709

### CODE REVIEW

**Comment by Manuel-777** [comment]
One issue I noticed with the overlays; When settings are updated (click on any setting, move trough tabs) some times the overlays are displayed even when there is no active overlay. Causing the window to be irresponsive (overlay covers the screen and is invisible).

To see this I enabled DevTools on the overlays and selected `body`. Then went into settings page, clicked some section and see how body now covers everything with no overlay elements. Enabling an overlay does not allow clicking trough the transparent areas, the only fix is disabling the overlay so settings are updated and the overlay is minimized.

**Comment by Thaoden** [comment]
Leave #684 out of this for a while, I'll try to port it onto the current dev as a new PR.

**Comment by Manuel-777** [comment]
Also, it seems Sort filter in decks page (and other pages) is not working properly since #665 @Lego6245 
Tried fixing it but only broke it further :man_facepalming:

**Comment by Lego6245** [comment]
See #705 for fixes.

File: src/shared/dom-fns.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  return parentNode.querySelectorAll(selectors);
+}
+
+export function queryElementsByClass(selectors:string, parentNode = document):any {
+  return parentNode.getElementsByClassName(selectors);
+}
+
+type returnableElements = HTMLDivElement | HTMLImageElement | HTMLInputElement | HTMLLabelElement | HTMLSpanElement;
+
+function createElement(
```
Comment (Thaoden) [inline src/shared/dom-fns.ts]: You can constrain the return type to `returnableElements` here, as you return the element passed in. nit: Maybe name the function differently, as it does not _create_ but rather _enrich_(?) the element.

Comment (Thaoden) [reply src/shared/dom-fns.ts]: Alternatively, pull the `document.createElement()` from the callers into this function and just pass in the html item string.

Comment (Thaoden) [reply src/shared/dom-fns.ts]: Should / can `attrs` be typed?

**Pull request merged**